### ISSUE TITLE
Ability to use this library for "img src" tags

### ISSUE THREAD

**Issue by pliablepixels** [silkimen/cordova-plugin-advanced-http#81]
Hi there, thanks for this library. I am currently facing an issue where I am working with a backend (3rd party) server for home surveillance. It sends camera frames as JPEG images over a TCP connection. So in my client, if I do `<img src="server/cameraX/feed">` it keeps streaming live feed as a continuous flow. My app is based on cordova and chrome.

The problem, of course, is Chrome only allows a max of 6 connections   to a domain and the only way around this is ugly subdomain or port changing hacks which I want to avoid. 

Is it possible to somehow use this library instead of chrome's HTTP implementation? The problem here is that the image streaming is continuous. `<img src>` in HTML handles this automatically. If I try and do an `http.get`, the get will keep going and not stop (as the server keeps sending images).  Even if I did chunk it up on the backend, it looks like I have to convert to base64 to display, which is a huge memory hog, especially if you assume many cameras displayed together.

So bottom line, is it possible to some how use `img src` and use this library's http implementation instead in the backend?

thanks

**Comment by silkimen** [comment]
Hi pliablepixels, unfortunately I think it's not possible to do. You'd need to change the source code of chrome and I don't know if this is possible in Cordova. Even if it is possible, you won't need this plugin.

Another possibility, maybe, is to write some kind of native "proxy" which is handling the requests and returning every single frame as a picture. Just an idea... don't know how to handle though.

Which format exactly are you using for streaming? Is it MJPEG?

**Comment by pliablepixels** [comment]
@silkimen thank you very much for your response. Ah, this was my last hope ;)
I believe the [server basically sends JPEG images over multi part mime](https://github.com/ZoneMinder/zoneminder/blob/ac5d1738eaec2ff5ac0e74bf371aab5f8e799eb6/src/zm_monitor.cpp#L3557-L3560) - that is probably MPJEG...
